Add Uppaal SMC metamodel.

ennoruijters opened the pull request:
We propose to include our metamodel for Uppaal-SMC in your repository as well. We refer to it in the SETTA 2017 paper, although we don't describe it in much detail there.

ennoruijters committed:
Add Uppaal SMC metamodel.

ramononis commented on metamodel/nl.utwente.ewi.fmt.uppaalsmc/src/nl/utwente/ewi/fmt/uppaalSMC/Serialization.java:
```diff
@@ -0,0 +1,147 @@
+			for (Selection s : e.getSelection()) {
+				if (!first) {
+					ret.append(", ");
+					first = false;
```
Move ```first = false;``` below the if-block.

ennoruijters committed:
Minor changes.

Almost entirely the result of autogenerating the sources with a newer
version of Ecore. I also removed some OCL constraints that were
indicated as syntax errors in Eclipse.
